src/core/sequence_state.cc

```diff
@@ -69,6 +69,26 @@ SequenceState::RemoveAllData()
   return Status::Success;
 }
 
+Status
+SequenceState::SetStringDataToZero()
+{
+  if (Data()->TotalByteSize() % 4 != 0) {
+    return Status(
+        Status::Code::INVALID_ARG,
+        "Incorrect data size for setting the string data to zero.");
```

Comment:
Maybe report a more verbose error saying that the size must be a multiple of 4 or why the size in invalid?